pt-liferay-solutions/liferay-portal
# LPS-140168 & LPS-146482 & LPS-146483 & LPS-146484 & LPS-146485 & LPS-146594 & LPS-146636 [CP 2.0] Exporting Activation Keys - DXP

### How it was developed?

It was developed a management activation keys bar, with an action drop down button to export all activation keys option.
For this moment, this button just have one option. Some others options will be added in future.
If download goes successfully it will show a success alert.
If download fails it will show a danger alert.

**Drop down button component**:

![Screenshot from 2022-02-03 19-19-06](https://user-images.githubusercontent.com/77743987/152438729-d2327ff9-327d-4384-ae1c-e7adbc3165f2.png)


### What has been affected by the development ?

- DXPActivationKeysTable: management activation keys bar was inserted inside, and a little style classes were adjusted/removed

### Adding data mock up to test

Nothing was added on data mock up.


### Test Plan

1. Enter in a project with DXP.
2. Click on action button.
3. Click at export activation keys option, this should download the file.
4. If download works fine, see if a success alert appears, if not, see if a fail alert appears.


PS: Make sure that you're authenticated with Okta sign in. (User Garrett)
